meson.build

```diff
@@ -145,7 +145,7 @@ configh_data.set('WIN32_LEAN_AND_MEAN', 1)
 # Supports -Wl,--version-script?
 have_version_script = cc.links(
     'int main(){}',
-    args: '-Wl,--version-script=' + meson.current_source_dir()/'xkbcommon.map',
+    args: '-Wl,--undefined-version,--version-script=' + meson.current_source_dir()/'xkbcommon.map',
```

Comment:
Certainly it is a bug if libxkbcommon depends on linker options that only work with specific bleeding-edge versions of binutils.

I am surprised that this broken PR was ever merged, at least in the state it was in. This is an lld-specific issue, so it should have been gated on `cc.get_linker_id() == 'lld'`.

Reply:
Let’s focus on this issue in #481.